IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,127 @@
+// Sprint Challenge: Swift Fundamentals - Ice Cream Shop
+// August Oliver
+// May 8th, 2020
+
+
+import Foundation
+
+//struct Flavor {
```

Comment:
While it's useful to comment out code because you may need it later, it's best to remove this when you're submitting your project.